tests/RobotFramework/tests/tedge_agent/device_profile/device_profile_initialization.robot

```diff
@@ -0,0 +1,24 @@
+*** Settings ***
+Resource            ../../../resources/common.resource
+Library             ThinEdgeIO
+Library             Cumulocity
+Library             OperatingSystem
+Library             Collections
```

Comment:
are the Libraries OperatinSystem and Collections needed?

Reply:
Removed the unnecessary libraries.

Reply:
Yeah copy paste stuff